__docs__/wonder-blocks-tabs/responsive-tabs-testing-playtesting.stories.tsx

```diff
@@ -0,0 +1,313 @@
+import {Meta, StoryObj} from "@storybook/react-vite";
+import {expect, fn, waitFor, within} from "storybook/test";
+import {ResponsiveTabs} from "@khanacademy/wonder-blocks-tabs";
+import {
+    Default as ResponsiveTabsDefault,
+    INITIAL_TABS_COUNT,
+} from "./responsive-tabs.stories";
+
+export default {
+    title: "Packages / Tabs / ResponsiveTabs / Testing / Responsive Tabs - Playtesting",
+    parameters: {
+        /** These stories are used for testing purposes only so we disable snapshots */
+        chromatic: {disableSnapshot: true},
+        a11y: {
+            config: {
+                rules: [
+                    // Disabling a11y violation: ""Elements must meet minimum
+                    // color contrast ratio thresholds (color-contrast)".
+                    // This is because the playtesting stories include switching
+                    // between layouts with a fade in animation. Because of this,
+                    // the a11y check will fail on an in between state where the
+                    // color is not yet updated to the final color. Note: This
+                    // a11y check is fully enabled for other ResponsiveTabs
+                    // stories.
+                    {
+                        id: "color-contrast",
+                        enabled: false,
+                    },
```

Comment:
I was getting some flaky results around color contrast in the playtesting stories. The detected foreground color would be a color that's not in our design system (it'd be a lighter version of gray or blurple). I believe it's because there is a fade in animation, and it was doing the a11y check during that animation so the color was not fully set yet. I disabled the color checks for the playtesting stories, however, they are enabled and passing in the other ResponsiveTabs stories. We don't see these a11y errors in the a11y panel too when running the checks manually

Let me know if there's another approach to handle the flakiness. I couldn't find an option to delay the a11y checks!